src/job.rs

```diff
@@ -43,8 +44,14 @@ impl<'a> Job<'a> {
     }
 
     /// Return job body.
-    pub fn body(&self) -> &str {
-        self.body.as_str()
+    pub fn body(&self) -> &[u8] {
+        &self.body[..]
+    }
+
+    /// Return job body as UTF-8 `&str`  
+    /// This method is just calling `std::str::from_utf8(&self.body)`
+    pub fn body_utf8(&self) -> BeanstalkcResult<&str> {
```

Comment:
Hi, maybe we can just remove `body_utf8`? `str:from_utf8(job.body().to_vec())` or `str::from_utf8_lossy(job.body())` just works. User can choose a proper method to decode the job payload. :)